recipe_scrapers/americastestkitchen.py

```diff
@@ -84,6 +87,10 @@ def category(self):
     def ratings(self):
         return self.schema.ratings()
 
+    @staticmethod
+    def _strip_html(string):
+        return normalize_string(re.sub(r"<.*?>", "", string))
```

Comment:
This seems useful - we may want to promote this into the `recipe_scrapers._utils` module at some point (cc @jknndy re: some similar code in `tasteau`, `tasteatlas`)

Reply:
Just opened #1117 with this change, only a few test case updates but seems like an improvement overall!